internal/cmd/load-balancer/describe/describe.go

```diff
@@ -160,37 +158,17 @@ func renderListeners(listeners []loadbalancer.Listener) string {
 	for i := range listeners {
 		listener := listeners[i]
 		table.AddRow(*listener.Name, *listener.Port, *listener.Protocol, *listener.TargetPool)
+		table.AddSeparator()
```

Comment:
If these tables are horizontal, I don't think we need the separators

Reply:
My point before was that when we have auto-merging on columns we usually have separator between rows that are not merged (e.g. see `stackit project member list`)

Reply:
That raises another point, do these 2 last tables need to be horizontal? Because the first one for the LB is vertical

Reply:
Nevermind, those can actually have multiple rows